src/Uno.UI.Runtime.Skia.Gtk/GTK/GtkApplicationExtension.cs

```diff
@@ -0,0 +1,50 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32;
+using Windows.UI.ViewManagement;
+using Windows.UI.Xaml;
+
+namespace Uno.UI.Runtime.Skia
+{
+	public class GtkApplicationExtension : IApplicationExtension
+	{
+		private readonly Windows.UI.Xaml.Application _owner;
+		private readonly Windows.UI.Xaml.IApplicationEvents _ownerEvents;
+
+		public GtkApplicationExtension(object owner)
+		{
+			_owner = (Application)owner;
+			_ownerEvents = (IApplicationEvents)owner;
+		}
+
+		public ApplicationTheme GetDefaultSystemTheme()
+		{
+			if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+			{
+				return GetWindowsTheme();
+			}
+			else
+			{
+				return Gtk.Settings.Default.ApplicationPreferDarkTheme ? ApplicationTheme.Dark : ApplicationTheme.Light;
+			}
+		}
+
+		private static ApplicationTheme GetWindowsTheme()
+		{
+			//var subKey = @"Software\Microsoft\Windows\CurrentVersion\Themes\Personalize";
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)